solutions/fully-configurable/main.tf

```diff
@@ -15,6 +15,13 @@ module "resource_group" {
   existing_resource_group_name = var.existing_resource_group_name
 }
 
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
```

Comment:
Try to avoid using this data lookup - you need to be an account admin to use it. You can get the account ID from the `ibm_resource_instance` output